crates/viewer/re_viewer_context/src/tensor/image_stats.rs

```diff
@@ -121,8 +121,8 @@ impl ImageStats {
         // ---------------------------
 
         let datatype = match image.format.pixel_format {
-            Some(PixelFormat::NV12 | PixelFormat::YUY2) => {
-                // We do the lazy thing here:
+            Some(_) => {
+                // We do the lazy thing here since we convert everything to RGB8 right now anyways.
```

Comment:
Maybe it would be smart to prepare ourself for HDR though?

```suggestion
            Some(pixel_format) => {
                let max_value = pixel_format.max_component_value();
                …
```

Reply:
Unfortunately this isn't about the max component value of the pixel format but rather what we show which right now is a bit different. max_component_value would be an _interesting_ one also for limited range YUV stuff :thinking:
I'm leaving an additional note on this for the next person!